examples/trials/mnist-pytorch/adl.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM pytorch/pytorch:1.4-cuda10.1-cudnn7-runtime
+
+COPY mnist.py .
+COPY requirements.txt .
```

Comment:
didn't find the requirement.txt file

Reply:
https://github.com/microsoft/nni/blob/master/examples/trials/mnist-pytorch/requirements.txt

Reply:
ok.